README.md

```diff
@@ -62,6 +63,7 @@ astarte_ignore_ssl = false
 # gRPC connection to the Astarte Message Hub
 [astarte.grpc]
 endpoint = "http://[::1]:50051"
+node_id = "ASTARTE_MSGHUB_NODE_ID_HERE"
```

Comment:
```suggestion
#node_id = "ASTARTE_MSGHUB_NODE_ID_HERE"
```

Reply:
Do you think the stream node id should be optional also when retrieving the configuration from the config.toml?

Reply:
O yes, I missed the Deserialize bound on the builder. I think an overridable serde(default = "...") would be more user-friendly